vars/Debian.yml

```diff
@@ -21,3 +21,5 @@ __php_apc_conf_filename: 20-apcu.ini
 __php_opcache_conf_filename: 05-opcache.ini
 __php_fpm_daemon: php5-fpm
 __php_fpm_pool_conf_path: "/etc/php5/fpm/pool.d/www.conf"
+
+php_fpm_listen: "unix://var/run/php5-fpm.sock"
```

Comment:
With a fresh Drupal VM install with `geerlingguy/ubuntu1404` using Nginx, the box provisions correctly but Nginx doesn't render the page - instead it returns a 502 Bad Gateway error.

I've managed to get around it for now by adding `php_fpm_listen: "unix://var/run/php5-fpm.sock"` into Drupal VM's config.yml and re-provisioning.